modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -0,0 +1,123 @@
+infrastructure logs, application logs, and audit logs. The retention policy, 
+configured using the `maxAge` parameter in the Cluster Logging Custom Resource 
+(CR), determines when to delete the Elasticsearch indices. 
+
+If you do not create a retention policy, logs are retained for seven days by
+default. If you do not specify a retention policy for all three log sources,
+logs from only the sources with a retention policy are stored. For example, 
+if you do not set a retention policy for audit logs, the audit logs will not
+be retained and there will be no *audit-* index in Elasticsearch or Kibana. 
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+To configure the log retention time:
+
+. Edit the Cluster Logging CR to add or modify the `retentionPolicy` parameter:
++
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+...
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    elasticsearch:
+      nodeCount: 3
+    retentionPolicy: <1>
+      application:
+        maxAge: 1d
+      infra:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
+...
+----
+<1> Specify the time that Elasticsearch should retain each log source. Enter an 
+integer and a time designation: weeks(w), hours(h/H), minutes(m) and seconds(s). 
+For example, `1d` for one day. Logs older than the `maxAge` are deleted. 
+By default, logs are retained for seven days. 
+
+. You can verify the settings in the Elasticsearch Custom Resource (CR).
++
+For example, the following Elasticsearch CR has a retention policy to roll over 
+active indices for the infrastructure logs every eight hours and the old indices 
+are deleted seven days after rollover. {product-title} checks every 15 minutes 
+to determine if the infrastructure indices need to be rolled over.
++
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "Elasticsearch"
+metadata:
+  name: "elasticsearch"
+spec:
+  indexManagement:
+    policies: <1>
+      - name: infra-policy
+        phases:
+          delete:
+            minAge: 7d <2>
+          hot:
+            actions:
+              rollover:
+                maxAge: 8h <3>
+        pollInterval: 15m <4>
+...
+----
+<1> For each log source, the retention policy indicates when to delete and 
+rollover logs for that source. The policy includes: 
+<2> When the rolled-over indices will be deleted after rollover. This setting 
+is the `maxAge` you set in the cluster logging CR.
+<3> When to rollover the indices. This value is determined from the `maxAge` 
+you set in the cluster logging CR.  
+<4> When to check if rollover should be performed. This setting is the default 
+and cannot be changed.
++
+[NOTE]
+====
+Modifying the Elasticsearch CR is not supported. All changes to the retention 
+policies must be made in the cluster logging CR.
+==== 
++
+The Elasticsearch Operator creates or modifies the `indexmanagement-scripts` 
```

Comment:
@sichvoge do you think we need this much detail? IMO we could remove any mention of these scripts